src/shared/components/Settings/Account/MyAccount/index.jsx

```diff
@@ -696,9 +682,9 @@ export default class MyAccount extends React.Component {
                           <div styleName="field col-2">
                             <div styleName="validation-bar" className="form-field">
                               <div styleName={`password toggle-password ${focus['re-new-password-input'] ? 'focus' : ''}`}>
-                                <input id="re-new-password-input" styleName="password-input" ref={this.reNewPasswordRef} onChange={this.checkPassword} onFocus={this.onPasswordFocus} onBlur={this.onPasswordBlur} name="re-password" type={passwordInputType['new-password-input']} placeholder="TYPE YOUR NEW PASSWORD" minLength="8" maxLength="64" required />
+                                <input id="re-new-password-input" styleName="password-input" ref={this.reNewPasswordRef} onChange={this.checkPassword} onFocus={this.onPasswordFocus} onBlur={this.onPasswordBlur} name="re-password" type={passwordInputType['re-new-password-input']} placeholder="TYPE YOUR NEW PASSWORD" minLength="8" maxLength="64" required />
                                 <label htmlFor="newPasswordCheckbox" styleName="passwordCheckbox">
```

Comment:
Might want to change the `htmlFor` as well?

Reply:
@sumitdaga

Reply:
@ThomasKranitsas  
oh yes i was meaning to but forgot ...i have done it now!
but any idea why it is not working ?

Reply:
what do you mean is not working?

Reply:
when you click on the label the respective input element should focus right ?
it does not ...not just on this page but on other pages too ..  
for ex Add new device form in Tools > Devices